src/components/Map/Layers/AddSymbol.jsx

```diff
@@ -0,0 +1,107 @@
+import { useContext, useEffect } from "react";
+import MapContext from "../MapContext";
+import { Style, Icon, Fill, Stroke, Circle, Image, Text } from "ol/style";
+import Feature from "ol/Feature";
+import Point from "ol/geom/Point";
+import { fromLonLat, get } from "ol/proj";
+
+const AddCircles = (d) => {
+  var iconStyle = [
+    new Style({
+      image: new Circle({
+        radius: d.style.radius,
+        fill: new Fill({
+          color: d.style.fill,
+        }),
+        stroke: new Stroke({
+          color: d.style.stroke,
+          width: d.style.strokeWidth,
+        }),
+      }),
+    }),
+    new Style({
+      text: new Text({
+        text: d.name,
+        offsetY: 2,
+        scale: 2,
+        fill: new Fill({
+          color: "#black",
+        }),
+      }),
+    }),
+  ];
+
+  let features = d.data.map((item) => {
+    let feature = new Feature({
+      geometry: new Point(fromLonLat([item.lon, item.lat])),
+    });
+    feature.setStyle(iconStyle);
+    return feature;
+  });
+
+  return features;
+};
+
+const AddSymbols = (d) => {
```

Comment:
```suggestion
const addSymbols = (d) => {
```

Function names should start with lowercase character.